### PR TITLE
#556 move functions from admin panel

### LINKED ISSUE #556: [TECH DEBT] Shortener and feature switches admin panels have functions defined within the body of the components
## Brief description:
Functions should be moved out of the components

### PR DESCRIPTION
**Pull request by Pastedko**
Fixed the issue with edit and delete functionalities on the feature switches. Moved the functions nested inside the components outside for the feature switches and url shortener.

resolves #556 

## How to verify:

- [ ] Start the React frontend, Golang backend, Python backend.

- [ ] Open your browser or API client and head to /admin/dashboard/shortener.

- [ ] Test the functionality by creating a shortened url, editing it and deleting it.

- [ ] Check the code in packages/web/src/components/admin_page/feature_switch_page

- [ ] Open your browser or API client and head to /admin/dashboard/fswitches.

- [ ] Test the functionality by creating a new feature switch, editing it and deleting it.

- [ ] Check the code in packages/web/src/components/admin_page/url_shortener

### CODE REVIEW

File: packages/web/src/components/admin_page/feature_switch_page/render_switch.jsx [hunk 1]
```diff
@@ -7,50 +7,54 @@ import { FsContext } from '../../../feature_switches';
...
+    console.log('hello2');
+    const { name, value } = e.target;
+    setNewSwitch(prevState => ({
+        ...prevState,
+        [name]: name === 'is_enabled' ? value === 'true' : value,
+    }));
+};
+
+const handleSubmit = (e, newSwitch, onUpdate) => {
+    console.log('hello');
```
Comment (asynchroza) [inline packages/web/src/components/admin_page/feature_switch_page/render_switch.jsx:20]: Leftover log

File: packages/web/src/components/admin_page/feature_switch_page/render_switch.jsx [hunk 2]
```diff
@@ -7,50 +7,54 @@ import { FsContext } from '../../../feature_switches';
 import toast from 'react-hot-toast';
 import styles from './featureSwitch.module.css';
 
+const handleChange = (e, setNewSwitch) => {
+    console.log('hello2');
```
Comment (asynchroza) [inline packages/web/src/components/admin_page/feature_switch_page/render_switch.jsx:11]: Leftover log

File: packages/web/src/components/admin_page/url_shortener/table.jsx [hunk 3]
```diff
@@ -112,7 +120,13 @@ const UrlsTable = () => {
 
     return (
         <>
-            <OverlayTrigger show={showAddOverlay} placement="bottom" overlay={popover(onUpdate, errorMessage)}>
+            <OverlayTrigger
+                show={showAddOverlay}
+                placement="bottom"
+                overlay={popover(
+                    data => onUpdate(data, triggerFetch, errorMessage, setErrorMessage, setShowAddOverlay),
```
Comment (asynchroza) [inline packages/web/src/components/admin_page/url_shortener/table.jsx:127]: Nice. I like this, but some people might have problems reading this. 

I suggest that you add a comment on top of explaining how this anonymous function works.

**Comment by asynchroza** [comment]
Overall, impressive work for a first ticket. Good job, Tedo!

**Comment by asynchroza** [comment]
@Aleks1908 please verify that everything is working

**Review by Aleks1908 — APPROVED** [review]
Very good Tedko, keep up the good work

**Pull request merged**